yt_dlp/extractor/nebula.py

```diff
@@ -186,28 +175,58 @@ class NebulaIE(NebulaBaseIE):
         {
             'url': 'https://watchnebula.com/videos/money-episode-1-the-draw',
             'only_matching': True,
+        }, {
+            'url': 'https://nebula.tv/videos/tldrnewseu-did-the-us-really-blow-up-the-nordstream-pipelines',
+            'info_dict': {
+                'id': 'e389af9d-1dab-44f2-8788-ee24deb7ff0d',
+                'ext': 'mp4',
+                'display_id': 'tldrnewseu-did-the-us-really-blow-up-the-nordstream-pipelines',
+                'title': 'Did the US Really Blow Up the NordStream Pipelines?',
+                'description': 'md5:b4e2a14e3ff08f546a3209c75261e789',
+                'upload_date': '20230223',
+                'timestamp': 1677144070,
+                'channel': 'TLDR News EU',
+                'channel_id': 'tldrnewseu',
+                'uploader': 'TLDR News EU',
+                'uploader_id': 'tldrnewseu',
+                'uploader_url': r're:https://nebula\.(tv|app)/tldrnewseu',
+                'duration': 524,
+                'channel_url': r're:https://nebula\.(tv|app)/tldrnewseu',
+                'series': 'TLDR News EU',
+                'thumbnail': r're:https?://.*\.jpeg',
+                'creator': 'TLDR News EU',
+                '_old_archive_ids': ['nebula 63f64c74366fcd00017c1513'],
+            },
+            'params': {'skip_download': 'm3u8'},
         },
         {
             'url': 'https://beta.nebula.tv/videos/money-episode-1-the-draw',
             'only_matching': True,
         },
     ]
 
-    def _fetch_video_metadata(self, slug):
-        return self._call_nebula_api(f'https://content.api.nebula.app/video/{slug}/',
-                                     video_id=slug,
-                                     auth_type='bearer',
-                                     note='Fetching video meta data')
-
     def _real_extract(self, url):
         slug = self._match_id(url)
-        video = self._fetch_video_metadata(slug)
-        return self._build_video_info(video)
+        url, smuggled_data = unsmuggle_url(url, {})
+        if smuggled_data.get('id'):
+            return {
+                'id': smuggled_data['id'],
+                'display_id': slug,
+                'title': '',
+                **self._extract_formats(slug),
+            }
+
+        return {
+            **self._extract_video_metadata(self._call_api(
+                f'https://content.api.nebula.app/video/{slug}',
+                slug, note='Fetching video metadata')),
+            **self._extract_formats(slug),
+        }
 
 
 class NebulaSubscriptionsIE(NebulaBaseIE):
     IE_NAME = 'nebula:subscriptions'
-    _VALID_URL = rf'{_BASE_URL_RE}/myshows'
+    _VALID_URL = rf'{_BASE_URL_RE}/(?P<id>myshows)'
     _TESTS = [
         {
             'url': 'https://nebula.tv/myshows',
```

Comment:
This URL no longer exists. Instead, `https://nebula.tv/library` is used for a general library, with `https://nebula.tv/library/latest-videos` showing the latest videos.

The code currently still works, but the website now uses a new endpoint
```python
'https://content.api.nebula.app/video_episodes/?following=true&include=engagement&ordering=-published_at'
```